common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -47,14 +47,15 @@ def assert_enrollment_status(
             self,
             course_id=None,
             username=None,
+            email=None,
             expected_status=status.HTTP_200_OK,
             email_opt_in=None,
             as_server=False,
             mode=CourseMode.DEFAULT_MODE_SLUG,
             is_active=None,
             enrollment_attributes=None,
             min_mongo_calls=0,
-            max_mongo_calls=0,
+            max_mongo_calls=1,  # see student.models.py::1235
```

Comment:
Probably, a more meaningful comment should be left here. I mean, what if a`student.models.py` changes and the line number will be different.

Reply:
I want to remove that comment at all

Reply:
I guess I left a similar comment somewhere; sorry if I missed your reply

Reply:
Ah ok)

Reply:
comment was deleted